.code-samples.meilisearch.yaml

```diff
@@ -162,15 +162,128 @@ get_all_tasks_1: |-
             print(error)
         }
     }
-get_keys_1: |-
-    client.keys { (result: Result<Key, Swift.Error>) in
-        switch result {
-        case .success(let key):
-            print(key)
-        case .failure(let error):
-            print(error)
-        }
+get_on_key_1: |-
+  self.client.getKey(key: d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4) { result in
+      switch result {
+      case .success(let key):
+          print(key)
+      case .failure(let error):
+          print(error)
+      }
+  }
+get_all_keys_1: |-
+  self.client.getKeys() { result in
+      switch result {
+      case .success(let keys):
+          print(keys)
+      case .failure(let error):
+          print(error)
+      }
+  }
+create_a_key_1: |-
+  let keyParams = KeyParams(
+    description: "Add documents: Products API key",
+    actions: ["documents.add"],
+    indexes: ["products"],
+    expiresAt: "2042-04-02T00:42:42Z"
+  )
+  self.client.createKey(keyParams) { result in
+      switch result {
+      case .success(let key):
+          print(keys)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_a_key_1: |-
+  let keyParams = KeyParams(
+    description: "Add documents: Products API key",
+    actions: ["documents.add", "documents.delete"],
+    indexes: ["products", "reviews"],
+    expiresAt: "2042-04-02T00:42:42Z"
+  )
+  self.client.updateKey(
+      key: d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4
+      keyparams: keyParams
+  ) { result in
+      switch result {
+      case .success(let key):
+          print(keys)
+      case .failure(let error):
+          print(error)
+      }
+  }
+delete_a_key_1: |-
+  self.client.deleteKey(key: d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4)
+  { result in
+      switch result {
+      case .success:
+          print("success")
+      case .failure(let error):
+          print(error)
+      }
+  }
+
+security_guide_search_key_1: |-
+  client = MeiliSearch(host: "http://localhost:7700", apiKey: "apiKey")
+  client.index("patient_medical_record")
+    .search(parameters) { (result: Result<SearchResult<Record>, Swift.Error>) in
```

Comment:
You used the types here, but I think it is good to use the same idea in the whole document, with or without the types 😸

Reply:
Do you think it is best to add the types or avoid complexity as I guess the IDE of the user is supposed to help them know about the types

Reply:
I think we could remove the types in the docs, since the idea is to be reeeeally simple there. And we could let the types to the source code itself.

Reply:
see #257